tests/local-integration-test.sh

```diff
@@ -35,6 +35,16 @@ if [ -z "$GITHUB_TOKEN" ] || [ -z "$GOVC_URL" ]; then
     exit 1
 fi
 
+# check if govc env command works as expected
+if [[ $(govc env GOVC_URL_SCHEME) =~ GOVC ]]; then
```

Comment:
Could just check the latest release version is being used: govc version -require 0.9.0